rllib/agents/pg/README.md

```diff
@@ -0,0 +1,16 @@
+Policy Gradient (PG)
```

Comment:
Minmal overview of the algo in a readme file.

Reply:
This seems to mostly duplicate the documentation. Can we instead provide a hyperlink to the doc page?

Reply:
done